js/completeTodo.js

```diff
@@ -0,0 +1,32 @@
+import { allCheckBoxes, tasks } from "./todoapp.js";
+import storage from "./storage.js";
+
+function completeTodo() {
+  for (let checkbox of allCheckBoxes) {
```

Comment:
naming should be consistent

if you write `checkbox` then `allCheckboxes`

if you write `checkBox` then `allCheckBoxes`